src/app.js

```diff
@@ -53,8 +53,14 @@ if (urlBase && siteId) {
 }
 
 const render = () => {
+  const domainTag = process.env.DOMAIN_TAG
+  const tags = [
+    release && `frontend tag ${release}`,
```

Comment:
Purely esthetics: since you have a colon in the next line, put one here as well
```suggestion
    release && `frontend tag: ${release}`,
```